vignettes/scoring-rules.Rmd

```diff
@@ -0,0 +1,470 @@
+---
+title: "Scoring rules in `scoringutils`"
+author: "Nikos Bosse"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Scoring rules in `scoringutils`}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+bibliography: "scoring-rules/scoringutils-package.bib"
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  fig.width = 7,
+  collapse = TRUE,
+  comment = "#>",
+  cache = FALSE
+)
+library(scoringutils)
+library(data.table)
+```
+
+# Introduction
+
+This vignette gives an overview of the default scoring rules made available through the `scoringutils` package. You can, of course, also use your own scoring rules, provided they follow the same format. If you want to obtain more detailed information about how the pacakge works, have a look at the [revised version](https://drive.google.com/file/d/1URaMsXmHJ1twpLpMl1sl2HW4lPuUycoj/view?usp=drive_link) of our `scoringutils` paper. 
```

Comment:
its really weird to me that we keep pointing at this google drive. I think if we are going to do this we should add it to `inst` as a pdf and point at that for now